automatic/tor-browser/update.ps1

```diff
@@ -1,6 +1,6 @@
 ﻿import-module au
 
-$releases = "https://www.torproject.org/download/languages/"
+$releases = "https://archive.torproject.org/tor-package-archive/torbrowser/13.0.6/"
```

Comment:
Thanks for implementing my suggestion about the archive. New thoughts:
- the "releases" url is where we look for new versions, not where we download them
- the "baseUrl" is where we download
- don't hardcode a version number in either of them

My suggestion here: 
```
$releases = "https://www.torproject.org/download/languages/"
$baseUrl  = "https://archive.torproject.org/tor-package-archive/torbrowser"
```